examples/jenkins-image-sample.groovy

```diff
@@ -199,6 +199,8 @@ void actualTest() {*/
                     dcs.describe()
                     echo "DeploymentConfig history"
                     dcs.rollout().history()
+
+                    //openshift.verifyService('mongodb')
```

Comment:
Any reason to keep this commented out?

Reply:
we cannot actually add tests until *after* the pr merges and a new version of the plugin is created, otherwise it breaks our overnight tests

minor drawback with sharing the PR tests and overnight tests that we just live with